PlatformConfig.mk

```diff
@@ -74,15 +74,18 @@ BOARD_AVB_VBMETA_SYSTEM_ROLLBACK_INDEX_LOCATION := 1
 
 TARGET_NO_RECOVERY := true
 
-# https://source.android.com/devices/bootloader/partitions/generic-boot#combinations, "Launch device without recovery partition":
+# https://source.android.com/devices/bootloader/partitions/generic-boot#combinations, "Launch device with A/B recovery partition":
```

Comment:
Their title reads weird, as if the A/B'ness only applies to the recovery partition, and this system is oblivious to the rest?